### PR TITLE
[DSM] DDP-8877 SM ID fix for RC

### PR DESCRIPTION
**Pull request by pegahtah**
fixed the duplicate sm id message and the deleted ones showing
Screen Shot 2022-10-05 at 11 36 13 AM

### CODE REVIEW

File: ddp-workspace/projects/ddp-dsm-ui/src/app/tissue/tissue.component.html [hunk 1]
```diff
@@ -386,82 +386,87 @@
...
-              </md-hint>
-            </md-input-container>
-          </td>
-          <td *ngIf="!smid.deleted">
-            <button md-mini-fab color="warn" [disabled]="!canChangeThis(i, this.currentSMIDField)"
-                    (click)="deleteSMID(this.tissue.HESMId, i)">
-              <i class="fas fa-trash-alt fa-sm"></i>
-            </button>
-          </td>
+          <ng-container *ngIf="!smid.deleted">
```
Comment (pegahtah) [inline ddp-workspace/projects/ddp-dsm-ui/src/app/tissue/tissue.component.html:389]: the only change is this line

File: ddp-workspace/projects/ddp-dsm-ui/src/app/tissue/tissue.component.html [hunk 2]
```diff
@@ -386,82 +386,87 @@
...
-              </md-hint>
-            </md-input-container>
-          </td>
-          <td  *ngIf="!smid.deleted">
-            <button md-mini-fab color="warn" [disabled]="!canChangeThis(i, this.currentSMIDField)"
-                    (click)="deleteSMID(this.tissue.ussSMId, i)">
-              <i class="fas fa-trash-alt fa-sm"></i>
-            </button>
-          </td>
+          <ng-container *ngIf="!smid.deleted">
```
Comment (pegahtah) [inline ddp-workspace/projects/ddp-dsm-ui/src/app/tissue/tissue.component.html:418]: the only change is this line

File: ddp-workspace/projects/ddp-dsm-ui/src/app/tissue/tissue.component.html [hunk 3]
```diff
@@ -386,82 +386,87 @@
...
-            </md-input-container>
-          </td>
-          <td  *ngIf="!smid.deleted">
-            <button md-mini-fab color="warn" [disabled]="!canChangeThis(i, this.currentSMIDField)"
-                    (click)="deleteSMID(this.tissue.scrollSMId, i)">
-              <i class="fas fa-trash-alt fa-sm"></i>
-            </button>
-          </td>
-
+            <ng-container *ngIf="!smid.deleted">
```
Comment (pegahtah) [inline ddp-workspace/projects/ddp-dsm-ui/src/app/tissue/tissue.component.html:447]: the only change is this line

File: ddp-workspace/projects/ddp-dsm-ui/src/app/tissue/sm-id.model.ts [hunk 4]
```diff
@@ -10,7 +10,7 @@ export class TissueSmId {
   ) {}
 
   static parse( json ): TissueSmId {
-    return new TissueSmId( json.smIdPk, json.smIdType, json.smIdValue, json.tissueId, false );
+    return new TissueSmId( json.smIdPk, json.smIdType, json.smIdValue, json.tissueId, json.deleted );
```
Comment (pegahtah) [inline ddp-workspace/projects/ddp-dsm-ui/src/app/tissue/sm-id.model.ts:13]: the deleted fix

**Review by SimoneMaiwald — APPROVED** [review]

**Pull request merged**